src/app/screens/swap/swapConfirmation/useConfirmSwap.tsx

```diff
@@ -52,17 +52,17 @@ export function useConfirmSwap(input: SwapConfirmationInput): SwapConfirmationOu
   const [unsignedTx, setUnsignedTx] = useState<StacksTransaction>(
     deserializeTransaction(input.unsignedTx),
   );
-  const [feeAmount, setFeeAmount] = useState(input.lpFeeAmount);
-  const [feeFiatAmount, setFeeFiatAmount] = useState(input.lpFeeFiatAmount);
+  const [feeAmount, setFeeAmount] = useState(input.txFeeAmount);
+  const [feeFiatAmount, setFeeFiatAmount] = useState(input.txFeeFiatAmount);
 
   return {
     ...input,
-    lpFeeAmount: isSponsored ? 0 : feeAmount,
-    lpFeeFiatAmount: isSponsored ? 0 : feeFiatAmount,
+    txFeeAmount: isSponsored ? 0 : feeAmount,
+    txFeeFiatAmount: isSponsored ? 0 : feeFiatAmount,
```

Comment:
```suggestion
    txFeeAmount: feeAmount,
    txFeeFiatAmount: feeFiatAmount,
```

can remove this, as we now hide the whole Fee block if sponsored